src/main/java/mate/academy/utils/ConnectionUtil.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.utils;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final String DRIVER_PATH = "com.mysql.cj.jdbc.Driver";
+    private static final String DRIVER_LOAD_FAILURE_MESSAGE = "Driver loading failure!";
+    private static final String PROPERTY_USER = "user";
+    private static final String PROPERTY_PASSWORD = "password";
+    private static final String DB_USERNAME = "root";
+    private static final String DB_PASSWORD = "QmZo_25107";

```

Comment:
Generally it is very bad practice to hardcode your credentials. Probably better option would be to fetch them from env variables, what do you think?